hack/update-apidocs.sh

```diff
@@ -44,7 +44,8 @@ EOF
 EOF
 
   find ./pkg/apis/kubeone/"${version}" -name '*types.go' -print0 |
-    xargs go run ./hack/apidoc-gen/main.go -section-link="#${version}" >> "${docfile}"
+    xargs -0 go run ./hack/apidoc-gen/main.go -section-link="#${version}" >> "${docfile}"
```

Comment:
This is required because `xargs` throws some not-so-nice warnings otherwise:

```
xargs: WARNING: a NUL character occurred in the input.  It cannot be passed through in the argument list.  Did you mean to use the --null option?
```